manifests/mod/ssl.pp

```diff
@@ -11,6 +11,12 @@
     'freebsd' => '/var/run/ssl_scache(512000)',
   }
 
+  $stapling_cache = $::osfamily ? {
+    'debian'  => "\${APACHE_RUN_DIR}/ocsp(128000)",
+    'redhat'  => '/var/cache/mod_ssl/ocsp(128000)',
+    'freebsd' => '/var/run/ocsp(128000)',
```

Comment:
Im failing to understand why the cache value would cause issues
with strict_variables.

Please note that the cache value i have used is the apache default. It can
be tweaked.

On 11 February 2015 at 23:39, Morgan Haskel <notifications@github.com>
wrote:

> In manifests/mod/ssl.pp
> <https://github.com/puppetlabs/puppetlabs-apache/pull/907#discussion_r24536089>
> :
>
> > @@ -11,6 +11,12 @@
> >      'freebsd' => '/var/run/ssl_scache(512000)',
> >    }
> >
> > +  $stapling_cache = $::osfamily ? {
> > +    'debian'  => "\${APACHE_RUN_DIR}/ocsp(128000)",
> > +    'redhat'  => '/var/cache/mod_ssl/ocsp(128000)',
> > +    'freebsd' => '/var/run/ocsp(128000)',
>
> Is there any reasonable value we can set as a default case? Otherwise this
> will cause issues with strict_variables.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetlabs-apache/pull/907/files#r24536089>
> .
>